authors/apps/PasswordResetToken/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.1.2 on 2018-10-31 16:56
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
```

Comment:
We agreed not to commit the migration files. You can undo this by running
```bash
 git rm authors/apps/PasswordResetToken/migrations/0001_initial.py
```

Reply:
Cool